src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/GregtechMetaTileEntity_LargeRocketEngine.java

```diff
@@ -89,9 +89,12 @@ public GregtechMetaTileEntity_LargeRocketEngine(final String aName) {
     protected GT_Multiblock_Tooltip_Builder createTooltip() {
         GT_Multiblock_Tooltip_Builder tt = new GT_Multiblock_Tooltip_Builder();
         tt.addMachineType(getMachineType()).addInfo("Controller Block for the Large Rocket Engine")
+                .addInfo("Generating Power from Rocket Fuels - Supports TecTech Multi-Amp Dynamos!")
                 .addInfo("Supply GT++ Rocket Fuels and 1000L of " + mLubricantName + " per hour")
                 .addInfo("Produces as much energy as you put fuel in, with optional boosting")
+                .addInfo("This multi sometimes doesn't accept fluids if not enabled - enable it first!")
```

Comment:
"sometimes" is phrased a bit awkwardly. We can be more precise here, something like "Fluid input can be toggled in the GUI", (I actually don't know if it's in the GUI as I'm unfamiliar with this multi, but I hope you get the idea), or "If this multi doesn't accept fluids, make sure input is enabled".